webpack.config.node.js

```diff
@@ -65,7 +66,14 @@ module.exports = {
 			{
 				test: /\.jsx?$/,
 				exclude: /(node_modules|devdocs\/search-index)/,
-				loader: 'babel-loader'
+				loader: 'babel',
+				query: {
+					plugins: [
+						[ 'transform-wpcalypso-async', {
```

Comment:
We don't want multiple bundles being generated though, in any case, so can we set `async` to `false`?